include/deal.II/matrix_free/matrix_free.h

```diff
@@ -640,6 +640,13 @@ class MatrixFree : public Subscriptor
    */
   //@{
   /**
+   * Returns whether a given FiniteElement @p fe is supported by this class.
+   */
+  template <int spacedim>
+  static
+  bool supports_fe (const FiniteElement<dim, spacedim> &fe);
```

Comment:
> But it may look less obvious in the class documentation

there one has extra doxygen which would clarify this.

anyway, I also don't feel strongly about either naming convection.